styleguide/source/assets/scss/03-organisms/_product-nav.scss

```diff
@@ -0,0 +1,14 @@
+.ama__product-nav {
+  background: $hoverBlack;
+
+  li {
+    display: inline-block;
+
+    a {
```

Comment:
Can we use the class ama__link--white here?

Reply:
I'm now including it via the .json file.